<?xml version="1.0"?>
<launch>
    <!-- 启动服务端 -->
    <node name="gripper_server" 
            pkg="gripper_control" 
            type="gripper_server.py" 
            output="screen"/>

    <!-- 在新终端中启动客户端交互界面 -->
    <node name="gripper_client" 
            pkg="gripper_control" 
            type="gripper_client.py"
            launch-prefix="gnome-terminal -- bash -c 'python3 $(find gripper_control)/scripts/gripper_client.py; echo; read -p &quot;按回车键关闭窗口...&quot;'"
            output="screen"/>
      <include file="$(find arm_control)/launch/arm_all.launch"/>
</launch>